content/app/development/ux/logo/_index.en.md

```diff
@@ -1,6 +1,7 @@
 ---
-title: Use organisation logo
-description: How to configure use of organisation logo
+title: Logo
+description: How to configure use of organization logo
```

Comment:
Kanskje endre `description` til "How to set and configure the logo of an application" e.l.?